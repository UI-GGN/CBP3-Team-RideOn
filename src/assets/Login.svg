<svg width="819" height="454" viewBox="0 0 819 454" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0_2_864)">
<path d="M214.737 142.221C240.642 142.221 261.642 121.233 261.642 95.3435C261.642 69.454 240.642 48.4664 214.737 48.4664C188.832 48.4664 167.831 69.454 167.831 95.3435C167.831 121.233 188.832 142.221 214.737 142.221Z" fill="#FF6584"/>
<path d="M361.827 239.802H131.723C131.435 239.805 131.147 239.783 130.863 239.734L239.692 51.3405C240.356 50.1672 241.32 49.1912 242.485 48.512C243.65 47.8329 244.975 47.4751 246.324 47.4751C247.673 47.4751 248.997 47.8329 250.162 48.512C251.328 49.1912 252.292 50.1672 252.956 51.3405L325.993 177.767L329.492 183.816L361.827 239.802Z" fill="#3F3D56"/>
<path opacity="0.2" d="M361.827 239.801H283.264L321.83 183.816L324.606 179.783L325.994 177.767L329.493 183.816L361.827 239.801Z" fill="black"/>
<path d="M489.947 239.802H290.926L329.492 183.816L332.268 179.783L382.523 106.824C385.818 102.042 393.744 101.744 397.618 105.922C397.878 106.207 398.119 106.508 398.341 106.824L489.947 239.802Z" fill="#3F3D56"/>
<path d="M548.368 208.809C556.174 208.809 562.502 202.484 562.502 194.683C562.502 186.882 556.174 180.557 548.368 180.557C540.562 180.557 534.234 186.882 534.234 194.683C534.234 202.484 540.562 208.809 548.368 208.809Z" fill="#1976D2"/>
<path d="M549.73 237.23H546.835L548.155 192.555L549.73 237.23Z" fill="#3F3D56"/>
<path d="M548.41 201.023L551.518 196.725L548.367 202.086L548.027 201.491L548.41 201.023Z" fill="#3F3D56"/>
<path d="M548.07 205.362L544.962 201.065L548.112 206.426L548.452 205.83L548.07 205.362Z" fill="#3F3D56"/>
<path d="M678.022 208.809C685.828 208.809 692.156 202.484 692.156 194.683C692.156 186.882 685.828 180.557 678.022 180.557C670.216 180.557 663.888 186.882 663.888 194.683C663.888 202.484 670.216 208.809 678.022 208.809Z" fill="#1976D2"/>
<path d="M679.384 237.23H676.489L677.81 192.555L679.384 237.23Z" fill="#3F3D56"/>
<path d="M678.065 201.023L681.173 196.725L678.022 202.086L677.681 201.491L678.065 201.023Z" fill="#3F3D56"/>
<path d="M677.725 205.362L674.616 201.065L677.767 206.426L678.107 205.83L677.725 205.362Z" fill="#3F3D56"/>
<path d="M589.312 195.254C600.841 195.254 610.187 185.914 610.187 174.392C610.187 162.87 600.841 153.529 589.312 153.529C577.783 153.529 568.437 162.87 568.437 174.392C568.437 185.914 577.783 195.254 589.312 195.254Z" fill="#1976D2"/>
<path d="M591.324 237.23H587.049L588.997 171.249L591.324 237.23Z" fill="#3F3D56"/>
<path d="M589.375 183.754L593.965 177.408L589.312 185.326L588.809 184.446L589.375 183.754Z" fill="#3F3D56"/>
<path d="M588.872 190.164L584.282 183.818L588.934 191.735L589.438 190.855L588.872 190.164Z" fill="#3F3D56"/>
<path d="M640.219 195.254C651.748 195.254 661.094 185.914 661.094 174.392C661.094 162.87 651.748 153.529 640.219 153.529C628.69 153.529 619.344 162.87 619.344 174.392C619.344 185.914 628.69 195.254 640.219 195.254Z" fill="#1976D2"/>
<path d="M642.231 237.23H637.956L639.905 171.249L642.231 237.23Z" fill="#3F3D56"/>
<path d="M640.282 183.754L644.872 177.408L640.219 185.326L639.716 184.446L640.282 183.754Z" fill="#3F3D56"/>
<path d="M639.779 190.164L635.189 183.818L639.842 191.735L640.345 190.855L639.779 190.164Z" fill="#3F3D56"/>
<path d="M508.178 195.254C519.707 195.254 529.053 185.914 529.053 174.392C529.053 162.87 519.707 153.529 508.178 153.529C496.649 153.529 487.303 162.87 487.303 174.392C487.303 185.914 496.649 195.254 508.178 195.254Z" fill="#1976D2"/>
<path d="M510.19 237.23H505.915L507.864 171.249L510.19 237.23Z" fill="#3F3D56"/>
<path d="M508.241 183.754L512.831 177.408L508.178 185.326L507.676 184.446L508.241 183.754Z" fill="#3F3D56"/>
<path d="M507.739 190.164L503.148 183.818L507.801 191.735L508.304 190.855L507.739 190.164Z" fill="#3F3D56"/>
<path d="M755.149 454H73.9061C65.323 454 56.8239 452.31 48.8942 449.028C40.9645 445.745 33.7593 440.934 27.6902 434.868C21.621 428.803 16.8067 421.602 13.5221 413.677C10.2375 405.752 8.54694 397.258 8.54694 388.68C8.54694 380.102 10.2375 371.608 13.5221 363.683C16.8067 355.758 21.621 348.558 27.6902 342.492C33.7593 336.427 40.9645 331.615 48.8942 328.333C56.8239 325.05 65.323 323.36 73.9061 323.36H753.641C760.308 323.36 766.702 320.714 771.416 316.002C776.13 311.291 778.779 304.901 778.779 298.237C778.779 291.574 776.13 285.184 771.416 280.473C766.702 275.761 760.308 273.114 753.641 273.114H51.2818V232.918H753.641C770.975 232.918 787.599 239.8 799.857 252.049C812.114 264.299 819 280.914 819 298.237C819 315.561 812.114 332.176 799.857 344.426C787.599 356.675 770.975 363.557 753.641 363.557H73.9061C67.239 363.557 60.845 366.204 56.1307 370.916C51.4164 375.627 48.7679 382.017 48.7679 388.68C48.7679 395.343 51.4164 401.733 56.1307 406.445C60.845 411.156 67.239 413.803 73.9061 413.803H755.149V454Z" fill="#E6E6E6"/>
<path d="M107.591 252.011H76.4199V254.021H107.591V252.011Z" fill="white"/>
<path d="M196.077 252.011H164.906V254.021H196.077V252.011Z" fill="white"/>
<path d="M284.564 252.011H253.392V254.021H284.564V252.011Z" fill="white"/>
<path d="M373.05 252.011H341.878V254.021H373.05V252.011Z" fill="white"/>
<path d="M461.536 252.011H430.365V254.021H461.536V252.011Z" fill="white"/>
<path d="M550.022 252.011H518.851V254.021H550.022V252.011Z" fill="white"/>
<path d="M638.508 252.011H607.337V254.021H638.508V252.011Z" fill="white"/>
<path d="M726.995 252.011H695.823V254.021H726.995V252.011Z" fill="white"/>
<path d="M107.591 342.454H76.4199V344.464H107.591V342.454Z" fill="white"/>
<path d="M196.077 342.454H164.906V344.464H196.077V342.454Z" fill="white"/>
<path d="M284.564 342.454H253.392V344.464H284.564V342.454Z" fill="white"/>
<path d="M373.05 342.454H341.878V344.464H373.05V342.454Z" fill="white"/>
<path d="M461.536 342.454H430.365V344.464H461.536V342.454Z" fill="white"/>
<path d="M550.022 342.454H518.851V344.464H550.022V342.454Z" fill="white"/>
<path d="M638.508 342.454H607.337V344.464H638.508V342.454Z" fill="white"/>
<path d="M726.995 342.454H695.823V344.464H726.995V342.454Z" fill="white"/>
<path d="M107.591 432.897H76.4199V434.907H107.591V432.897Z" fill="white"/>
<path d="M196.077 432.897H164.906V434.907H196.077V432.897Z" fill="white"/>
<path d="M284.564 432.897H253.392V434.907H284.564V432.897Z" fill="white"/>
<path d="M373.05 432.897H341.878V434.907H373.05V432.897Z" fill="white"/>
<path d="M461.536 432.897H430.365V434.907H461.536V432.897Z" fill="white"/>
<path d="M550.022 432.897H518.851V434.907H550.022V432.897Z" fill="white"/>
<path d="M638.508 432.897H607.337V434.907H638.508V432.897Z" fill="white"/>
<path d="M726.995 432.897H695.823V434.907H726.995V432.897Z" fill="white"/>
<path d="M546.893 79.7628C550.168 79.7597 553.39 80.5916 556.253 82.1798H537.558C540.413 80.5921 543.626 79.7601 546.893 79.7628Z" fill="#E6E6E6"/>
<path d="M510.616 77.3457C516.086 77.3393 521.424 79.0282 525.893 82.1798H495.359C499.823 79.0319 505.152 77.3432 510.616 77.3457Z" fill="#E6E6E6"/>
<path d="M305.12 46.7481C305.783 35.0241 310.591 23.9167 318.687 15.4061C326.783 6.89557 337.64 1.53512 349.322 0.280485C361.005 -0.974146 372.754 1.9586 382.474 8.55585C392.194 15.1531 399.254 24.986 402.395 36.3014C402.995 36.2799 403.595 36.2558 404.201 36.2558C414.913 36.2578 425.336 39.7257 433.911 46.1409C442.486 52.5561 448.752 61.5735 451.771 71.8443C458.869 68.7275 466.801 68.0512 474.324 69.9215C481.848 71.7918 488.538 76.103 493.348 82.1798H338.694C334.182 82.1926 329.714 81.2953 325.558 79.5418C321.401 77.7883 317.641 75.2145 314.503 71.9746C311.365 68.7348 308.914 64.8954 307.295 60.6863C305.677 56.4773 304.925 51.9852 305.085 47.4789C305.095 47.2357 305.106 46.9921 305.12 46.7481Z" fill="#E6E6E6"/>
<path d="M558.178 115.94C554.904 115.937 551.682 116.769 548.819 118.357H567.514C564.659 116.769 561.446 115.937 558.178 115.94Z" fill="#E6E6E6"/>
<path d="M594.456 113.523C588.986 113.516 583.648 115.205 579.178 118.357H609.713C605.249 115.209 599.919 113.52 594.456 113.523Z" fill="#E6E6E6"/>
<path d="M799.952 82.9253C799.288 71.2013 794.48 60.0938 786.384 51.5833C778.289 43.0727 767.432 37.7122 755.749 36.4576C744.067 35.203 732.318 38.1357 722.598 44.733C712.878 51.3302 705.818 61.1631 702.677 72.4786C702.076 72.457 701.476 72.4329 700.87 72.4329C690.159 72.4349 679.736 75.9029 671.161 82.318C662.586 88.7332 656.32 97.7507 653.3 108.021C646.202 104.905 638.271 104.228 630.747 106.099C623.224 107.969 616.533 112.28 611.724 118.357H766.378C770.89 118.37 775.357 117.472 779.514 115.719C783.671 113.965 787.431 111.392 790.569 108.152C793.707 104.912 796.158 101.072 797.776 96.8634C799.395 92.6544 800.147 88.1623 799.987 83.656C799.977 83.4128 799.965 83.1692 799.952 82.9253Z" fill="#E6E6E6"/>
<path d="M696.846 425.91L714.82 425.919L714.822 423.068L696.847 423.058L696.846 425.91Z" fill="#3F3D56"/>
<path d="M566.889 423.702L570.597 424.559L711.54 424.634L712.807 421.683C715.018 416.621 715.776 411.045 714.999 405.577C714.392 401.501 712.87 397.451 709.416 395.692L703.161 354.058L625.185 353.919L595.016 378.237C595.016 378.237 580.785 377.982 573.053 387.601C570.264 391.178 568.618 395.511 568.328 400.036L568.023 405.04L566.889 423.702Z" fill="#1976D2"/>
<path d="M597.845 440.399C607.457 440.399 615.249 432.612 615.249 423.005C615.249 413.399 607.457 405.612 597.845 405.612C588.233 405.612 580.441 413.399 580.441 423.005C580.441 432.612 588.233 440.399 597.845 440.399Z" fill="#3F3D56"/>
<path d="M597.845 432.16C602.904 432.16 607.005 428.061 607.005 423.005C607.005 417.95 602.904 413.851 597.845 413.851C592.786 413.851 588.685 417.95 588.685 423.005C588.685 428.061 592.786 432.16 597.845 432.16Z" fill="#CCCCCC"/>
<path d="M683.723 440.445C693.335 440.445 701.127 432.657 701.127 423.051C701.127 413.445 693.335 405.658 683.723 405.658C674.111 405.658 666.319 413.445 666.319 423.051C666.319 432.657 674.111 440.445 683.723 440.445Z" fill="#3F3D56"/>
<path d="M683.723 432.206C688.782 432.206 692.883 428.107 692.883 423.051C692.883 417.995 688.782 413.897 683.723 413.897C678.664 413.897 674.563 417.995 674.563 423.051C674.563 428.107 678.664 432.206 683.723 432.206Z" fill="#CCCCCC"/>
<path d="M607.002 375.395L649.103 375.413H654.243L662.919 375.419V374.787L662.926 367.161L662.932 358.599H657.159L652.019 358.593L642.916 358.587H637.777L627.267 358.58L607.002 375.395Z" fill="white"/>
<path d="M669.77 375.426L692.02 375.438L692.026 371.074V363.449L692.032 358.617H688.774L683.635 358.611L669.776 358.605L669.77 375.426Z" fill="white"/>
<path d="M660.058 393.384L662.911 393.386L662.914 387.968L660.061 387.967L660.058 393.384Z" fill="#3F3D56"/>
<path d="M618.408 383.097L618.41 380.246L612.989 380.243L612.988 383.094L618.408 383.097Z" fill="#3F3D56"/>
<path d="M616.412 381.499L616.099 381.499C615.315 381.499 614.54 381.344 613.817 381.044C613.093 380.744 612.436 380.305 611.883 379.751C611.33 379.198 610.891 378.541 610.591 377.817C610.292 377.094 610.138 376.319 610.139 375.537C610.139 374.754 610.294 373.979 610.594 373.256C610.894 372.533 611.333 371.877 611.887 371.324C612.441 370.77 613.099 370.332 613.823 370.033C614.546 369.734 615.322 369.58 616.105 369.58L616.419 369.581L616.412 381.499Z" fill="#3F3D56"/>
<path d="M568.023 405.04C570.18 405.275 572.351 404.741 574.154 403.532C575.956 402.324 577.274 400.519 577.875 398.435C578.476 396.35 578.323 394.121 577.44 392.139C576.558 390.157 575.005 388.55 573.054 387.601C570.265 391.178 568.618 395.511 568.328 400.036L568.023 405.04Z" fill="#3F3D56"/>
<path d="M652.02 358.593L662.919 374.787L662.926 367.161L657.159 358.599L652.02 358.593Z" fill="#F2F2F2"/>
<path d="M683.635 358.611L692.026 371.074V363.449L688.774 358.617L683.635 358.611Z" fill="#F2F2F2"/>
<path d="M637.777 358.586L649.103 375.413H654.243L642.916 358.586H637.777Z" fill="#F2F2F2"/>
<path d="M46.2541 287.183C71.7996 287.183 92.5083 266.487 92.5083 240.957C92.5083 215.427 71.7996 194.731 46.2541 194.731C20.7087 194.731 0 215.427 0 240.957C0 266.487 20.7087 287.183 46.2541 287.183Z" fill="#3F3D56"/>
<path d="M46.4466 254.874L23.2994 231.741C22.5465 230.986 22.1239 229.964 22.1245 228.899C22.125 227.834 22.5488 226.812 23.3026 226.059C24.0564 225.305 25.0786 224.882 26.1446 224.881C27.2106 224.881 28.2333 225.303 28.9879 226.056L46.0617 243.119L100.545 181.027C101.249 180.227 102.242 179.739 103.305 179.67C104.369 179.601 105.416 179.956 106.217 180.659C107.018 181.361 107.508 182.352 107.579 183.415C107.649 184.478 107.295 185.525 106.593 186.327L46.4466 254.874Z" fill="#1976D2"/>
</g>
<defs>
<clipPath id="clip0_2_864">
<rect width="819" height="454" fill="white"/>
</clipPath>
</defs>
</svg>
