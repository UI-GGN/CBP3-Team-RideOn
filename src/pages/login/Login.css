.logo {
  margin-left: 70px;
  margin-top: 40px;
}


.login-page-body {
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  align-items: center;
  margin-top: 170px;
}

.body-left {
  width: 37rem;

.body-right {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  font-family: Nunito;
  font-style: normal;
  letter-spacing: 0.07rem;
  width: 34rem;
}

.login-heading-text {
  font-weight: 700;
  font-size: 1.5rem;
  line-height: 2.5rem;
}

.login-body-text {
  font-weight: 500;
  font-size: 1.2rem;
  line-height: 2.2rem;
}
