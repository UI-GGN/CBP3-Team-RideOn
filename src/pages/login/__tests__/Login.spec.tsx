import {render} from "@testing-library/react";
import Login from "../Login.js";

describe("Login test", () => {
  it("renders correctly", () => {
    const {baseElement} = render(<Login />);
    expect(baseElement).toMatchSnapshot();
  });
});
