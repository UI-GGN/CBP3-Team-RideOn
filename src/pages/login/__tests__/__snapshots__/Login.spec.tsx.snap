// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login test renders correctly 1`] = `
<body>
  <div>
    <div>
      <img
        class="logo"
        src="Logo.png"
      />
      <div
        class="login-page-body"
      >
        <div>
          <img
            class="body-left"
            src="Login.png"
          />
        </div>
        <div
          class="body-right"
        >
          <div>
            <h3
              class="login-heading-text"
            >
              What is RideOn?
            </h3>
            <p
              class="login-body-text"
            >
              Simplifying employee cab management for seamless transportation logistics. Effortlessly manage employee transportation with RideOn, the ultimate cab management system.
            </p>
          </div>
          <div>
            <p
              class="login-body-text"
            >
              To book track or manage, click the button below
            </p>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge css-jh47zj-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              LOGIN WITH GOOGLE
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
